app/lib/main.dart

```diff
@@ -172,56 +150,99 @@ class _EffektioHomeState extends State<EffektioHome> {
                         margin: const EdgeInsets.only(bottom: 10, right: 10),
                         child: Icon(Icons.search),
                       ),
-                      onPressed: () {
-                        setState(() {});
-                      },
+                      onPressed: () {},
                     )
                   ],
                 ),
-          body: _widgetOptions.elementAt(tabIndex),
+          body: TabBarView(
+            controller: _tabController,
+            children: [
+              NewsScreen(
+                client: client,
+              ),
+              FaqOverviewScreen(client: client),
+              NewsScreen(
+                client: client,
+              ),
+              ChatList(client: _client),
+              NotificationScreen(),
+            ],
+          ),
           drawer: SideDrawer(
             client: _client,
           ),
-          bottomNavigationBar: Container(
-            decoration: BoxDecoration(
-              boxShadow: [
-                BoxShadow(color: Colors.grey, offset: Offset(0, -0.5)),
-              ],
+          bottomNavigationBar: TabBar(
+            labelColor: AppCommonTheme.primaryColor,
+            unselectedLabelColor: AppCommonTheme.svgIconColor,
+            controller: _tabController,
+            indicator: BoxDecoration(
```

Comment:
Isn't this customizable? The actual indicator appears as short and a bit rounded from edges.

Reply:
![Screenshot (52)](https://user-images.githubusercontent.com/68579938/177376804-851fc345-4d91-4f17-b324-10420ede15df.png)